jenkins-pipelines/perf-regression-latency-125gb-with-ebs-volume.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    aws_region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: "test-cases/performance/perf-regression-latency-125gb-with-ebs-volume.yaml",
```

Comment:
Why this is not using "configuration file" and the ebs is embedded inside the yaml?
To make it easiest for comparison, there should be a test latency 125gb regular and one that uses the configuration of ebs.

Reply:
We don't have jenkins-pipelines/perf-regression-latency-125gb.jenkinsfile such job and configuration files.
Then i will add them in new PR, and then rebase and fix this issues

Reply:
Why not in this PR so we can track it?